libs/newsletter/testing/src/drivers/testing/newsletter.service.ts

```diff
@@ -1,14 +1,15 @@
 import { DaffNewsletterSubmission } from '@daffodil/newsletter';
-import { Observable, of } from 'rxjs';
+import { Observable, of, BehaviorSubject, timer } from 'rxjs';
 import { Injectable } from '@angular/core';
 import { DaffNewsletterServiceInterface } from '@daffodil/newsletter';
+import { delay } from 'rxjs/operators';
 
 @Injectable({
   providedIn: 'root'
 })
 
 export class DaffTestingNewsletterService implements DaffNewsletterServiceInterface<DaffNewsletterSubmission, any>{
   send(payload: DaffNewsletterSubmission): Observable<any>{
-    return of('Success');
+    return of('Success').pipe(delay(100));
```

Comment:
Yes I think that would be a good idea

Reply:
To follow up on this, the delay is actually needed because the testing service will return the 
``` of('success')``` to quickly for the switchMap to cancel it.